src/components/Question-Card/question-card.component.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import './question-card.style.css';
+
+export const answerChecker = (wrong_id: number) => {
```

Comment:
Rather than directly manipulating the DOM, there are ways for you to do this using React itself e.g you make each answer to be a part of a list of components and add a "wrong-answer" className to any of those components when the user has selected an answer AND the correct answer is not equal to the answer in that component.